pkg/virt-controller/services/template.go

```diff
@@ -561,6 +562,11 @@ func getRequiredResources(vmi *v1.VirtualMachineInstance) k8sv1.ResourceList {
 	if (vmi.Spec.Domain.Devices.AutoattachPodInterface == nil) || (*vmi.Spec.Domain.Devices.AutoattachPodInterface == true) {
 		res[TunDevice] = resource.MustParse("1")
 	}
+	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
+		if iface.Driver == "vhost" {
```

Comment:
Do we want to introduce constants like `NetDriverVHost` and `NetDriverQEMU` which we can use in the converter and on the cluster side?

Reply:
That's reasonable. For me, `InterfaceDriverVhost` and `InterfaceDriverQEMU` seem more natural as enum members for `Interface`'s `Driver` if used on the cluster side.